dashboards/datadog_audit_trail_overview/README.md

```diff
@@ -0,0 +1,28 @@
+# Audit Trail Overview
+
+Understand how your end-users use Datadog by leveraging Audit Trail.
+
+![Dashboard Preview #1](./img/screenshot1.jpg)
+
+# How to use the dashboard
+
+Helps w/ understanding how end-users are using Datadog.
```

Comment:
A bit more of information could be really useful. How am I supposed to use this? Anything that I need to know beforehand?